Sources/SpeziHealthCharts/MetricChart/ChartViews/Modifiers/HealthChart+Style.swift

```diff
@@ -0,0 +1,22 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+extension EnvironmentValues {
+    @Entry var healthChartStyle: HealthChartStyle = .default
+}
+
+
+extension View {
+    // TODO: Add argument here to control how we combine styles (e.g. .override, .combine, etc.)
+    public func style(_ newValue: HealthChartStyle) -> some View {
```

Comment:
I think a naming like `healthChartStyle(_:)` would be more unique and fit great with existing SwiftUI APIs (like `buttonStyle(_:)` or `labelStye(_:)`, ...).